src/locale/index.js

```diff
@@ -2,10 +2,16 @@
 import * as C from '../constants';
 // /////////////////////////////////////////////////////////////////////////////////////////////////
 
-const UA = {
+export const UA = {
   termsAndConditions: 'Умови використання',
   privacyPolicy: 'Політика приватності',
   copyright: '2019 © Всі права захищено',
+  localeNames: {
+    [C.LOCALE_NAME_UA]: 'Українська',
+    [C.LOCALE_NAME_EN]: 'Англійська',
```

Comment:
Maybe better not to translate languages. After accident change of language user might be confused how to select previous language if he does not know how it looks on current